system/services/presideObjects/PresideObjectService.cfc

```diff
@@ -2947,6 +2951,25 @@ component displayName="Preside Object Service" {
 		return expanded;
 	}
 
+	private array function _getDefaultFilters( required string objectName, required array ignoreDefaultFilters ){
+		var adminRequest = $getRequestContext().isAdminRequest();
+		var defaultFilters = [];
+
+		if ( !adminRequest ){
```

Comment:
We should not assume that the default filters are not to be used in admin requests. For example, we may have an admin only application that wants to use default filters. 

If more fine-grained default filters want to be used, interceptors can already be used to achieve this.

Reply:
Hi @DominicWatson.

I had removed the admin checking. Please do let me know if there is anything else.

Thank you,
Malcolm